src/libawkward/forth/ForthMachine.cpp

```diff
@@ -1253,7 +1253,7 @@ namespace awkward {
 
   template <typename T, typename I>
   void
-  ForthMachineOf<T, I>::maybe_throw(util::ForthError err,
+  ForthMachineOf<T, I>::maybe_throw(util::ForthError,
```

Comment:
That's right: the error state is managed with the `current_error_` class member (because this is all single-threaded). The `err` argument to this function may be a relic.